### DISCUSSION TITLE
i wonder wot countries.json for

### DISCUSSION THREAD

**Discussion by tintinkung** [ASEAN-Build-The-Earth/Merlion#52] [Q&A] [answered]
> yes, what is this for 
>
> ![image](https://user-images.githubusercontent.com/77855014/137519224-4df85792-7a22-433e-9121-da3b19c43701.png)

**Comment by StoneMcYT** [comment, marked as answer]
https://medium.com/the-dev-caf%C3%A9/a-discord-bot-to-track-covid-19-c31d2288b8f7

**Reply by tintinkung** [reply]
woa

**Reply by tintinkung** [reply]
did we actually use it yet tho

**Reply by StoneMcYT** [reply]
> did we actually use it yet tho

I dont think so
